src/jobs/backend-tests.yml

```diff
@@ -0,0 +1,40 @@
+description: >
+  This job will automate running a set of backend tests.
+
+parameters:
+  # Not Required Parameters
+  docker-image:
+    type: string
+    default: "cimg/base:stable"
+    description: |
+      The docker image that will be used as an executor
+      if isNode is true, node/default is recommended
+  isNode:
```

Comment:
When we get to Python tests will we add an isPython parameter?

Reply:
Right, we can just expand it like that then just add the when condition and what is nessecary.